platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ImportMigrationConfigurationAdminEntry.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration.admin;
+
+import java.io.IOException;
+import java.util.Dictionary;
+import java.util.Enumeration;
+import java.util.Objects;
+
+import javax.annotation.Nullable;
+
+import org.apache.commons.lang.Validate;
+import org.codice.ddf.migration.ImportMigrationEntry;
+import org.codice.ddf.migration.ImportMigrationEntryProxy;
+import org.codice.ddf.migration.MigrationException;
+import org.osgi.framework.Constants;
+import org.osgi.service.cm.Configuration;
+import org.osgi.service.cm.ConfigurationAdmin;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class extends on the {@link ImportMigrationEntry} interface to represent an exported
+ * configuration object.
+ */
+public class ImportMigrationConfigurationAdminEntry extends ImportMigrationEntryProxy {
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            ImportMigrationConfigurationAdminEntry.class);
+
+    private final ImportMigrationConfigurationAdminContext context;
+
+    private final Dictionary<String, Object> properties;
```

Comment:
✏️ What about `importProperties` just to be clear and specific?

Reply:
Already inside the import entry. It seemed to me like it is obvious.

Reply:
No, because there are other places in this class that we're dealing with properties from the in-memory config object. My point is that we need to be clear about the difference between what came from the import and what's in the current system.